src/grt/grt-wave_opt_file-parse-debug.adb

```diff
@@ -39,22 +42,24 @@ package body Grt.Wave_Opt_File.Parse.Debug is
          else
             Put_Line ("Instances : ");
          end if;
-         Dump_Sub_Tree (Trees (Index), 1);
+         Dump_Sub_Tree (Trees (Index));
       end loop;
       Put_Line ("----------- END -----------------");
       New_Line;
    end Dump_Tree;
 
-   procedure Dump_Sub_Tree (Cursor : Elem_Acc; Level : Positive)
+-------------------------------------------------------------------------------
```

Comment:
I prefer not to impose an order between 'public' and 'private' subprograms as it is often clearer to interleave them (because you group related subprograms).  That's why I usually don't separate them.

I am maybe too picky on style convention and I know this is not the most important point.